src/overlays/actors/ovl_En_Knight/z_en_knight.h

```diff
@@ -12,16 +12,16 @@ typedef struct EnKnight {
     /* 0x144 */ s16 unk_144;
     /* 0x146 */ char unk_146[0x2];
     /* 0x148 */ s8 unk_148;
-    /* 0x149 */ char pad149[8];
+    /* 0x149 */ char unk_149[0x8];
     /* 0x151 */ s8 unk_151;
-    /* 0x152 */ char pad152[1];
+    /* 0x152 */ char unk_152[0x1];
     /* 0x153 */ u8 unk_153;
     /* 0x154 */ u8 unk_154;
-    /* 0x155 */ char pad155[0x2CB];
+    /* 0x155 */ char unk_155[0x2CB];
     /* 0x420 */ EnKnightActionFunc actionFunc;
     /* 0x424 */ char unk_424[0x25C];
     /* 0x680 */ u8 unk_680;
-    /* 0x681 */ char pad681[9];
+    /* 0x681 */ char unk_681[0x9];
```

Comment:
Wait are we removing pads for unk_ values now? Do we know they are values used by another actor in the Igos fight?

edit: also they are char

Reply:
This actor has yet to be decomped, so unsure if they really are pads so would prefer `unk` for the time being. Same reason why they stay `char`

Reply:
We have no idea what these are since this actor has not been decompiled yet. The struct is therefore the same as the original but with some of the variables identified. `pad` means explicitly that it is known not to be used, that has always been how it's been used in these projects.